### ISSUE TITLE
stylish formatter should report the total number of errors and warnings at the end

### ISSUE THREAD

**Issue by molant** [webhintio/hint#201]
Right now it is only after each resource but it will be nice to have it also at the end as a summary to know how many things need to be fixed:
![image](https://cloud.githubusercontent.com/assets/606594/25981919/34550722-368d-11e7-99a0-073ae786cbdf.png)
